Makefile

```
@@ -145,7 +145,7 @@ xenon:
 	--max-modules B \
 	--max-average A \
 	--ignore "data, tests, docs" \
-	--exclude "src/fidesctl/core/annotate_dataset.py,src/fidesctl/_version.py"
+	--exclude "src/fidesctl/core/annotate_dataset.py,src/fidesctl/_version.py,src/fidesctl/cli/__init__.py"
```

I really dislike that we're omitting code that we wrote from this check. To me, it's a code smell if our functions are failing it, and it should be resolved with a refactor. How do you feel about me creating a separate issue to make this line look like:
```suggestion
	--exclude "src/fidesctl/_version.py"
```
and handle any refactoring needed to get the Xenon CI check to pass?

---

that sounds good to me! I'll open an issue to get xenon passing on everything but the __version__ magic